<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The expressions defined here are used to exclude files/directories from the scan.  You can add/remove expressions to customize the scanning process.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFiles">
      <attribute name="title">
       <string>Files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="treeViewFiles">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="addFileButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>532</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="deleteFileButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDirs">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QTreeView" name="treeViewDirs">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="addDirButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>532</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="deleteDirButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRegistry">
      <attribute name="title">
       <string>Registry</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="4">
        <widget class="QTreeView" name="treeViewReg">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="addRegButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="deleteRegButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete the selected scan documents</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="styled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>532</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTip">
     <property name="text">
      <string>Tip:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblExpressionHintLink">
     <property name="text">
      <string>&lt;a href=&quot;#expressions&quot;&gt;how to understand and write expressions&lt;a&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
